src/net/sourceforge/kolmafia/textui/parsetree/ProxyRecordValue.java

```diff
@@ -78,27 +78,27 @@ public Value aref(final int index, final AshRuntime interpreter) {
     }
 
     if (rv instanceof Integer) {
-      return DataTypes.makeIntValue(((Integer) rv).intValue());
+      return DataTypes.makeIntValue((Integer) rv);
```

Comment:
You can modernize this even more by doing this:
```
    if (rv instanceof Integer iv) {
      return DataTypes.makeIntValue(iv);
    }
```
and so on for all the similar casts in this method.

Reply:
I like that idea.  Stand by.

Reply:
@Veracity0 - Would you please resolve if I got it right?  Thank you.

Reply:
That was all good.
There is one more instance in that same method just before your first change that could be fixed as well to eliminate a a cast.